ADA/git_databricks.qmd

```diff
@@ -0,0 +1,221 @@
+---
+  title: "Databricks and version control"
+---
+
+<p class="text-muted">
+
+Guidance for analysts on how to connect Databricks to Github and DevOps
+
+</p>
+
+------------------------------------------------------------------------
+
+## Why should I use git with Databricks?
+
+------------------------------------------------------------------------
+
+#### Easier collaboration
+
+------------------------------------------------------------------------
+
+When you're working on notebooks in Databricks without a git connection, they tend to be saved in your own personal Workspace. This means that they are only accessible by you, unless you share them. This has the potential to cause issues if you're on leave and someone needs to run any code you have stored in a notebook within Databricks. If the notebooks are stored in a Github or DevOps repo, your team can all access them by cloning the repo in Databricks.
```

Comment:
Would be tempted to expand on the potential issues a little: "... the potential to cause continuity issues - e.g. when you leave your current team, non-working days, annual leave etc, team members will not be able to access any of your code stored in your notebooks on Databricks."